modules/apm/src/main/java/org/elasticsearch/telemetry/apm/internal/metrics/LongCounterAdapter.java

```diff
@@ -41,14 +41,20 @@ public void incrementBy(long inc, Map<String, Object> attributes) {
         getInstrument().add(inc, OtelHelper.fromMap(attributes));
     }
 
-    private static class Builder extends AbstractInstrument.Builder<LongCounter> {
+    private static class Builder extends AbstractInstrument.Builder<LongUpDownCounter> {
         private Builder(String name, String description, String unit) {
             super(name, description, unit);
         }
 
         @Override
-        public LongCounter build(Meter meter) {
-            return Objects.requireNonNull(meter).counterBuilder(name).setDescription(description).setUnit(unit).build();
+        public LongUpDownCounter build(Meter meter) {
+            LongUpDownCounter counter = Objects.requireNonNull(meter)
+                .upDownCounterBuilder(name)
+                .setDescription(description)
+                .setUnit(unit)
+                .build();
+            counter.add(0L);
```

Comment:
According to the test I did this `add(0)` was not really required.

Reply:
strange.. it is not working for me without it.
just tested this again. I have to add `.add(0)` to have this emitted after registration (without any increments)
can you try repeating your test?